lib/adapters/amqp/subscriber.ts

```diff
@@ -3,7 +3,9 @@ import {ConfigAMQP} from "../../config";
 import {Message} from "../../messageFactory";
 import serviceDetails = require("../../support/serviceDetails");
 
-const _ = require("lodash");
+const clone = require("lodash.clone");
+const isObject = require("lodash.isobject");
+const isString = require("lodash.isstring");
```

Comment:
`typeof` is enough
Only one detail: 
```js
const str = new String("My string");
console.log(typeof str);          // it prints `object`
``` 
As I said, I didn't remove any _unnecessary_ `lodash` function if it was doing something else than the simplest thing. 
`Lodash` is ensuring that `new String` is also considered `string`. 
There is no point on keeping `lodash.isstring`.

Given that you agree, I'll remove _unnecessary_ functions asap.